spring-integration-file/src/main/java/org/springframework/integration/file/config/FileParserUtils.java

```diff
@@ -118,7 +124,7 @@ static void configureFilter(BeanDefinitionBuilder synchronizerBuilder, Element e
 			}
 			if (count != 1) {
 				parserContext.getReaderContext().error("at most one of 'filename-pattern', " +
-						"'filename-regex', 'filter' or 'filter-expression' is allowed on remote file inbound adapter",
+								"'filename-regex', 'filter' or 'filter-expression' is allowed on remote file inbound adapter",
```

Comment:
But the extra indent pushed it past 120 - why did you even touch this line? - do you have some "auto-format on save"